website/docs/docs/deploy/job-scheduler.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "The dbt Cloud job scheduler"
+id: "job-scheduler"
+sidebar_label: "Job scheduler"
+description: "The dbt Cloud Scheduler queues scheduled or API-triggered runs, before preparing the job to enter cloud data platform. Build observability into transformation workflows with the in-app scheduling, logging, and alerting." 
+tags: [scheduler]
+---
+
+The Scheduler is the backbone of running jobs in dbt Cloud, bringing power and simplicity to building data pipelines in both continuous integration and production contexts. The Scheduler frees teams from having to build and maintain their own infrastructure and ensures the timeliness and reliability of data transformations.
+
+The Scheduler enables both cron-based and event-driven execution of dbt commands in the user’s warehouse. Specifically, the Scheduler handles:
+
+- Cron-based execution of dbt Cloud jobs that run on a predetermined cadence
+- Event-driven execution of dbt Cloud CI jobs triggered by pull requests to the the dbt repo
+- Event-driven execution of dbt Cloud jobs triggered by API
+- Event-driven execution of dbt Cloud jobs manually triggered by a user to "Run Now"
+
+The Scheduler takes care of:
```

Comment:
hey @schottj , wondered if we can turn this into a paragraph as it looks a bit weird having bullets follow bullets. what do you think?

```suggestion

The Scheduler handles various tasks including queuing jobs, creating temporary environments to run the dbt commands required for those jobs, providing logs for debugging and remediation, and storing dbt artifacts for direct consumption/ingestion by the Metadata API.
```

![Screenshot 2023-05-03 at 15 30 19](https://user-images.githubusercontent.com/89008547/235947093-93bdcf08-1b59-456e-aa4f-ac38c88caa78.png)

Reply:
works for me!